mathesar_ui/src/systems/table-view/table-inspector/column/column-extraction/ExtractColumnsModal.svelte

```diff
@@ -300,10 +300,10 @@
           text={targetTableName
             ? $_('columns_removed_from_table_added_to_target', {
                 values: { count: $columns.length },
-              })
+              }) + ' A link column will be created to maintain the relationship between records.'
```

Comment:
When you finish putting all your strings into dict.json, make sure you don't have any post-translation string concatenation like this.